src/components/FilterBar.js

```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import { DatePicker, Space } from 'antd';
+import moment from 'moment';
+
+
+export default function FilterBar(onChange) {
+  const [startDate, setStartDate] = useState(moment())
+  const [dataFilter, setDataFilter] = useState([])
+
+  function onChange(date) {
+    setStartDate(date)
+    const newData = list.filter(item => moment(item.created_at.substring(0, 10)).isSame(date, 'date'))
+    console.log(JSON.stringify(newData), "filter data")
+    setDataFilter(newData)
+  }
+
+  const [list] = useState([]);
+
+  return (
+      <div>
+        <Space direction="vertical" size={12}>
+          <DatePicker onChange={onChange} defaultValue={moment()} />
```

Comment:
<DatePicker onChange={props.onChange} defaultValue={moment()} />

Reply:
it supposes to be <DatePicker onChange={props.onChange} defaultValue={moment()} />

Reply:
onChange={onChange}